doc/polyglossia.tex

```diff
@@ -44,14 +43,77 @@
 \def\Ie{\textit{I.e.,}\xspace}
 \def\etc{\@ifnextchar.{\textit{etc}}{\textit{etc.}\@\xspace}}
 
+%% Commands for documenting options
+\ExplSyntaxOn
```

Comment:
I don't think we need to introduce `\ExplSyntax` for what follows.

Reply:
May be, but I don't know a replacement for `\seq_use:Nnnn`.

Reply:
Please try. At least all the other commands do not need `l3 `at all.

Reply:
I think, expl3 is nothing evil or unsuitable in the code of a manual.
As you seem to fear it, I will avoid `\seq_use:Nnn` by reintroducing the necessary *or*s to the list of possible values.